README.md

```diff
@@ -87,7 +87,7 @@ java_war(
     data_path = "webapp",
     deps = [
         "//external:appengine/java/api",
-        "//external:javax/servlet/api",
+        "@io_bazel_rules_appengine//appengine:javax.servlet.api",
```

Comment:
why do we have that redirect and not point directly to @javax_servlet_api//:jar?